packages/gatsby/cache-dir/__tests__/static-entry.js

```diff
@@ -147,6 +189,42 @@ describe(`static-entry`, () => {
     })
   })
 
+  test(`onPreRenderHTML can filter out null value`, done => {
+    global.plugins = [setNullHeaderPlugin, checkNonEmptyHeadersPlugin]
+
+    StaticEntry(`/about/`, (_, html) => {
+      expect(html).toMatchSnapshot()
```

Comment:
do we need snapshot testing here if we assert head components directly? I would like avoid busy snapshots like that that, especially that I think this snapshot would pass right now without any changes in `static-entry` file, right?

Reply:
As it stands they do seem to be redundant. I will remove them.